mica-webapp/src/main/resources/_templates/datasets.ftl

```diff
@@ -151,9 +148,48 @@
 <!-- ./wrapper -->
 
 <#include "libs/scripts.ftl">
+<script>
```

Comment:
JS script is only acceptable at this place when it is simple. You should move this code to a separate template so that user does not break logic when changing page's template layout.